.github/workflows/release.yml

```diff
@@ -35,6 +35,27 @@ jobs:
           checksum_file=$(echo "$ARTIFACTS" | jq -r '.[] | select (.type=="Checksum") | .path')
           echo "::set-output name=hashes::$(cat $checksum_file | base64 -w0)"
 
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: git fetch --prune --unshallow
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+          check-latest: true
+      - uses: imjasonh/setup-ko@v0.6 # This installs the current latest release.
+      - uses: sigstore/cosign-installer@v2.8.1
```

Comment:
Do we need `COSIGN_EXPERIMENTAL=true` or does this grab a version that no longer needs it?

Reply:
Not sure I understand the question 🙃

Reply:
> Not sure I understand the question 🙃

`COSIGN_EXPERIMENTAL` was needed to enter the keyless signing flow, but that was recently removed since keyless signing went GA.

cosign-installer@v2.8.1 installs cosign